.github/workflows/node-ci.yml

```diff
@@ -117,6 +117,9 @@ jobs:
       with:
         exclude: .github/,.storybook/
 
+    - name: Pre-publish check
+      run: node utils/pre-publish-check-ci.js
```

Comment:
can you move this and the changeset check to above the build size check? The build size check changes what code is checked out and should always go last.